appengine/cloudsql/README.md

```diff
@@ -1,13 +1,25 @@
 # Cloud SQL sample for Google App Engine
 This sample demonstrates how to use [Cloud SQL](https://cloud.google.com/sql/) on Google App Engine
+
 ## Setup
-Before you can run or deploy the sample, you will need to create a [Cloud SQL instance](https://cloud.google.com/sql/docs/create-instance)
-1. Create a new user and database for the application. The easiest way to do this is via the [Google Developers Console](https://console.developers.google.com/project/_/sql/instances/example-instance2/access-control/users). Alternatively, you can use MySQL tools such as the command line client or workbench.
+Before you can run or deploy the sample, you will need to create a [Cloud SQL instance)](https://cloud.google.com/sql/docs/create-instance)
+
+1. Create a new user and database for the application. The easiest way to do this is via the [Google
+Developers Console](https://console.developers.google.com/sql/instances). Alternatively, you can use MySQL tools such as the command line client or workbench.
```

Comment:
Should be https://console.cloud.google.com/sql/instances Remember console.developers.google.com just has the API manager now and no cloud stuff.